c2pa-proposal.md

```diff
@@ -98,3 +98,104 @@ The only requirement for the Implicit Attestation to work within C2PA Eco-system
     4) If the signing key is revoked for any reason (e.g. due to expiry), then a new key to be provisioned, has to undergo the same "Explicit Attestation Protocol".
 
 Specific Explicit "Key Attestation" Protocol is out of scope of C2PA Specification.
+
+## Explicit Attestations
+
+Explicit Attestation allow a C2PA Claim Signer to undergo attestation with a local or remote
+Trust broker and introduce the attestation as part of C2PA Manifest.
+Compared to Implicit Attestation an Explicit Attestation offers additional guarantees, such as:
```

Comment:
Explicit attestations encode the security posture of a device at the time of asset creation, whereas implicit attestations involve keys that were previously provisioned.  Explicit attestations are required if the platform does not provide facilities to limit access to authorized claim generators running in authorized environments - a capability commonly called "sealing."  Explicit attestations are generally easier to implement securely, because access-management of the implicit attestation keys is not required.  However, if sealing-functions are available, both explicit and implicit attestations can provide similarly detailed statements of the platform's security posture.

In the current proposal, a manifest can contain more than one explicit attestation (for example, a TPM-based attestation of a "rich OS" and an SGX-enclave attestation of the claim generator), but only one implicit attestation signature is allowed.